internal/action/command.go

```diff
@@ -641,6 +645,9 @@ func (h *BufPane) ShowKeyCmd(args []string) {
 		return
 	}
 
+	// Convert possible received string of raw escape sequence
```

Comment:
No need to duplicate the comment 3 times either. :) I would move the comment above the function definition.